roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -18,25 +18,25 @@
   tags: ngi_reports
 
 - name: Unpack TeXLive
-  unarchive:
-    src: "{{ texLive_dest }}/src/{{ texLive_file }}"
-    dest: "{{ texLive_dest }}/src/"
-    copy: no
-    creates: "{{ texLive_dest }}/src/install-tl-*/LICENSE.TL"
-    mode: g+rw
+  command: "tar -xzf {{ texLive_dest }}/src/{{ texLive_file }} -C {{ texLive_dest }}/src/ --strip-components=1"
```

Comment:
Yea, everything is unpacked directly under {{ texLive_dest }}/src/ .
Version management under /src/ is pointless. Strip components moves the output.
I could make another folder to pack into (which would be another 3 lines), but it feels a bit silly.